lib/Mojo/Message.pm

```diff
@@ -97,10 +97,14 @@ sub fix_headers {
   # Content-Length or Connection (unless chunked transfer encoding is used)
   my $content = $self->content;
   my $headers = $content->headers;
+  my $websocket = $headers->sec_websocket_accept;
+  my $is_websocket = defined $websocket;
```

Comment:
This line does not appear to serve a real purpose.

Reply:
no. it was for clearer intent. feel free to suggest a diffent way